pandas/core/nanops.py

```diff
@@ -360,7 +360,7 @@ def _wrap_results(result, dtype, fill_value=None):
 
             result = tslibs.Timedelta(result, unit="ns")
         else:
-            result = result.astype("i8").view(dtype)
+            result = result.astype("m8").view(dtype)
```

Comment:
would making this `m8[ns]` break things?

Reply:
It shouldn't.  I've started a test run on my arm box with it.